kernel/nativevalues.mli

```diff
@@ -116,6 +116,8 @@ val accu_nargs : accumulator -> int
 val cast_accu : t -> accumulator
 [@@ocaml.inline always]
 
+val is_accu : t -> bool
```

Comment:
if you give it the type `'a -> bool`, you might be able to avoid some `Obj.magic` in the generated code, I guess.